back/src/domains/convention/use-cases/broadcast/BroadcastToPoleEmploiOnConventionUpdates.ts

```diff
@@ -41,10 +62,15 @@ export class BroadcastToPoleEmploiOnConventionUpdates extends TransactionalUseCa
     { convention }: WithConventionDto,
     uow: UnitOfWork,
   ): Promise<void> {
-    const agency = await uow.agencyRepository.getById(convention.agencyId);
-    if (!agency)
-      throw errors.agency.notFound({ agencyId: convention.agencyId });
-    if (agency.kind !== "pole-emploi")
+    const { agency, refersToAgency } = await getLinkedAgencies(
+      uow.agencyRepository,
+      convention,
+    );
+
+    if (
+      (agency.kind !== "pole-emploi" && !refersToAgency) ||
+      (refersToAgency && refersToAgency.kind !== "pole-emploi")
+    )
```

Comment:
```suggestion
    if (
      (!refersToAgency && agency.kind !== "pole-emploi") ||
      (refersToAgency && refersToAgency.kind !== "pole-emploi")
    )
```

Juste pour garder une cohérence entre les deux lignes.